module1/homework/grayscaleImages/compression.hpp

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include <array>
+#include <utility>
+#include <vector>
+
+namespace {
```

Comment:
Usually, we don't put unnamed namespaces in hpp files. It doesn't change the visibility of these expressions.
You can safely remove unnamed `namespace`